src/bus/peer.c

```diff
@@ -605,7 +615,7 @@ int peer_queue_reply(Peer *sender, const char *destination, uint32_t reply_seria
         return 0;
 }
 
-static int peer_broadcast_to_matches(PolicySnapshot *sender_policy, NameSet *sender_names, MatchRegistry *matches, MatchFilter *filter, uint64_t transaction_id, Message *message) {
+static int peer_broadcast_to_matches(PolicySnapshot *sender_policy, NameSet *sender_names, MatchRegistry *matches, uint64_t sender_id, MatchFilter *filter, uint64_t transaction_id, Message *message) {
```

Comment:
Now the argument order looks pretty random, or is there some logic behind it? I'd do something like `sender-id, sender-creds..., matches, message`

Reply:
Just followed peer_broadcast(), but changed now.